src/test/resources/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlasTest.josm.osm

```diff
@@ -105,4 +105,9 @@
     <tag k='name' v='parent' />
     <tag k='type' v='parent' />
   </relation>
+  <relation id='-41861' action='modify'>
+    <member type='relation' ref='-41860' role='child1' />
+    <tag k='name' v='super-parent' />
```

Comment:
super shallow?? ;p

Reply:
Dang it I forgot about that official terminology :P

Reply:
Dang it I forgot about that official terminology :P